euphony/src/androidTest/java/co/euphony/rx/EuRxManagerTest.kt

```diff
@@ -12,16 +13,14 @@ import org.junit.runner.RunWith
 class EuRxManagerTest {
 
     private val rxManager: EuRxManager = EuRxManager()
-    private val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+    private val context : Context =  ApplicationProvider.getApplicationContext<Context>()
 
     @Test
-    public fun testListenWithContext() {
-        val permissionStatus = appContext.checkPermission(
-            RECORD_AUDIO,
-            Process.myPid(),
-            Process.myUid()
+    public fun testListen() {
+        val permissionStatus = ContextCompat.checkSelfPermission(context,
+            RECORD_AUDIO
         )
-        val result = rxManager.listen(appContext)
+        val result = rxManager.listen()
         assertEquals(permissionStatus == PERMISSION_GRANTED, result)
```

Comment:
You implemented error handling, so we have to write test code about whether error is not occurred when calling `listen()`. So, I think below test code is more proper.

```kotlin
    @Test(expected = Test.None::class) /* no exception expected */
    fun withExceptionHandlingNoErrorOccurs() {
        val rxManager = EuRxManager()
        rxManager.listen()
    }
```

Also, `context` variable is not used, so we have to remove line16 and unused import